tests/output/sql/index.html

```diff
@@ -22,8 +22,8 @@
         <script src="../../shared_utils.js"></script>
         <script>
             runTestFiles([
-                "output_test.js",
-                "assert_test.js"
+                "assert_test.js",
```

Comment:
I rearranged the order to match the webpage order (assert, then output) as one way to encourage mocha not to flake out on the first test.